packages/devtools/src/schema/parser.ts

```diff
@@ -95,9 +97,12 @@ export class SchemaParser {
         // return type
       },
       [MapperKind.OBJECT_TYPE]: (type: GraphQLObjectType) => {
+        const directives = getDirectives(this.#schema, type)
+        const indices = this._parseIndices(directives)
```

Comment:
Indices can only be set on models so I think the logic should be different here, only trying to parse the indices after having identified the object is a model and throwing an error if the `@createIndex` directive is used on embedded objects.